.github/workflows/bsk_pr_checks.yml

```diff
@@ -177,6 +187,11 @@ jobs:
             CODE_SIGNING_ALLOWED=NO \
             | tee -a ios-build-log.txt \
             | xcbeautify --report junit --report-path . --junit-report-filename bsk-ios-unittests.xml
+            
+      - name: Send Unit Tests Status Pixel
+        if: always() && github.ref_name == 'main'
+        run: |
+          curl -fLSs -o /dev/null "https://improving.duckduckgo.com/t/m_ci_apple_bsk_main_unit_tests_ios_${{ steps.run-unit-tests-ios.outcome }}?workflow=${{ env.WORKFLOW_URL }}"
```

Comment:
We could put `https://improving.duckduckgo.com/t/m_ci_apple` in a repository variable for reuse. But I will follow up with more pixels related to CI so I can take care of it later 👍